frontend/src/components/CheckboxFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+
+import { FilterContainer, FilterRow, Sort } from "./FilterCommon";
+import { FilterHeader } from "./FilterHeader";
+
+const CheckboxFilterRow = styled(FilterRow)`
+  padding-left: 20px;
+`;
+
+const FilterRadioButton = styled.img`
+  height: 20px;
+  width: 20px;
+`;
+
+const PopupBodyText = styled(Sort)`
+  font-weight: 400;
+  font-size: 12px;
+  margin-left: 0;
+`;
+
+export type CheckboxFilterProps = {
+  title: string;
+  options: string[];
+  value: Set<number>;
+  setValue(val: Set<number>): void;
+};
+
+export const CheckboxFilter = (props: CheckboxFilterProps) => {
+  return (
+    <FilterContainer>
+      <FilterHeader title={props.title} />
+      {props.options.map((text, idx) => (
+        <CheckboxFilterRow key={idx}>
+          <FilterRadioButton
+            src={
+              props.value.has(idx) ? "/filled_filter_radio_button.svg" : "/filter_radio_button.svg"
```

Comment:
Would it be helpful to reuse the `CustomCheckboxRadio` from [here](https://github.com/TritonSE/USHS-Housing-Portal/blob/44c825e17a329b90484e7ee33328eb7370e11f86/frontend/src/components/ListingForm/CommonStyles.tsx#L68-L72)? Or do you still want to import the SVG for the checkbox?

Reply:
Good point, I will reuse that. I didn't know that component existed :sweat_smile: